pkg/cache/node_info_test.go

```diff
@@ -21,9 +21,9 @@ package cache
 import (
 	"testing"
 
+	"github.com/apache/incubator-yunikorn-scheduler-interface/lib/go/common"
```

Comment:
Check the sort order
This belongs to  the local imports below.

Reply:
Done